appliances/FELDSAM/5A435F05-412D-4FC9-930A-0E6295760A78.yaml

```diff
@@ -0,0 +1,83 @@
+--- 
+id: 5A435F05-412D-4FC9-930A-0E6295760A78
+name: "Vrouter Load Ballancer Alpine 3.5 - KVM"
+version: "1.0.1"
+publisher: "FELDSAM s.r.o."
+description: |-
+    ## Features
+    1. NAT iptables masquerading enabled by new context var `ETHx_VROUTER_GATEWAY = "YES"`
+    2. Keepalived email notifications:
+    	* `VROUTER_KEEPALIVED_NOTIFY_EMAIL` - required to enable this feature
+    	* `VROUTER_KEEPALIVED_FROM_EMAIL` - optional, if not defined emails are send from `NOTIFY_EMAIL` address
+    3. Keepalived can define sync interface using var `ETHx_VROUTER_KEEPALIVED_INF` - usefull if you need save some public IPv4 addresses or just want all vrrp traffix on private interface. *For saving addresses need integration in OpenNebula*
+    4. Keepalived Load Ballancing support
+    	* both NAT and DR (Direct Routing)
+    	* Single and Multi Port support using firewall marks
+    	* FTP Passive support using multiport, fwmarks and ip\_vs\_ftp module
+    	* Context vars in following format `VROUTER_LB0_DEV`, `VROUTER_LB0_SERVER0_IP`, `VROUTER_LB0_SERVER0_CHECK0_TYPE`
+    	* *Limitation of 9 LoadBallancers on one vrouter instance each with max 9 real servers. This can be enhanced in context scripts in future.*
+    5. Iptables and Keepalived IPv6 support - for this time just in dual stack IPv4+IPv6
+    
+    **There are prepared context vars in template, so you just need define right values**  
+    [Docs here](https://forum.opennebula.org/t/contribution-vrouter-enhacements-and-load-ballancing/3679)
+short_description: "Vrouter with load ballancing support based on Alpine linux 3.5"
+tags: 
+  - vrouter
+  - loadballancer
```

Comment:
Can you please fix the double Ls in balancing words (s/ballanc/balanc/) in tags, short_description and description?